cypress/e2e/tests/rbac/architect.test.ts

```diff
@@ -33,7 +33,9 @@ import { CredentialType, legacyPathfinder } from "../../types/constants";
 
 let stakeholders: Array<Stakeholders> = [];
 
-describe(["@tier3", "@rhsso"], "Architect RBAC operations", function () {
+describe(["@tier3", "@rhsso"], "Bug issue-2397: Architect RBAC operations", function () {
```

Comment:
Could you just use `Issue 2397` instead of bug? RPJ would try to pick issue-2397 as a jira ticket and break

Reply:
Done